acl/src/main/java/org/apache/rocketmq/acl/plain/PlainAccessValidator.java

```diff
@@ -47,7 +47,7 @@ public PlainAccessValidator() {
     @Override
     public AccessResource parse(RemotingCommand request, String remoteAddr) {
         PlainAccessResource accessResource = new PlainAccessResource();
-        accessResource.setWhiteRemoteAddress(remoteAddr);
+        accessResource.setWhiteRemoteAddress(remoteAddr.split(":")[0]);
```

Comment:
It is better to check before split.
If remoteAddr does not contains ':' , just set the original value, otherwise split it.